### ISSUE TITLE
1.21.2 and 1.21.3 builds crash on launch

### ISSUE THREAD

**Issue by iijj22gg** [Wurst-Imperium/Mo-Glass#46]
I'm unsure why I didn't create an issue for this earlier, and I guess the only reason I'm doing this now is because I have no idea how to solve build errors that occur if I try to build for earlier versions.

```log
[00:02:01] [Render thread/INFO]: [STDOUT]: Starting Mo Glass...
---- Minecraft Crash Report ----
// Sorry :(

Time: 2024-12-01 00:02:01
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'mo_glass' at 'net.wurstclient.glass.MoGlassInitializer'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.class_310.<init>(class_310.java:482)
	at knot//net.minecraft.client.main.Main.main(Main.java:248)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.ExceptionInInitializerError
	at knot//net.wurstclient.glass.MoGlass.initialize(MoGlass.java:20)
	at knot//net.wurstclient.glass.MoGlassInitializer.onInitializeClient(MoGlassInitializer.java:25)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399)
	... 9 more
Caused by: java.lang.NullPointerException: Block id not set
	at java.base/java.util.Objects.requireNonNull(Objects.java:259)
	at knot//net.minecraft.class_4970$class_2251.method_63505(class_4970.java:664)
	at knot//net.minecraft.class_4970.<init>(class_4970.java:110)
	at knot//net.minecraft.class_2248.<init>(class_2248.java:191)
	at knot//net.minecraft.class_2482.<init>(class_2482.java:46)
	at knot//net.wurstclient.glass.GlassSlabBlock.<init>(GlassSlabBlock.java:29)
	at knot//net.wurstclient.glass.MoGlassBlocks.<clinit>(MoGlassBlocks.java:37)
	... 12 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.class_310.<init>(class_310.java:482)

-- Initialization --
Details:
	Modules: 
Stacktrace:
	at knot//net.minecraft.client.main.Main.main(Main.java:248)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
```

**Comment by Alexander01998** [comment]
Hmm, that error message does look familiar. This is something I ran into in 24w45a and fixed in 0509e5bc5ccfc2fbc03579d9be2b4f14026594ed. Wasn't aware of older MC versions being affected too.

**Comment by Alexander01998** [comment]
Hey @iijj22gg, sorry for the delay. I've fixed the 1.21.2/3 branches now.

I'll release a 1.10.1 patch soon for anyone using those versions. Weird that nobody reported this sooner, but there's a test for it now so it won't go unnoticed again.
